components/signup.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>The Shoebox</title>
+    <link rel="icon" type="image/x-icon" href="../assets/icon.png">
+
+    <!-- stylesheets -->
+    <link rel="stylesheet" href="../css/component_styles/signup.css">
+
+    <!-- font awesome -->
+    <script src="https://kit.fontawesome.com/e413b10342.js" crossorigin="anonymous"></script>
```

Comment:
Consider importing this directly in your main root CSS file, allows you to automatically have it imported into your CSS components and HTML

Reply:
okh I will do it, 👍 thanks for the rest